data/json/recipes/recipe_others.json

```diff
@@ -3252,7 +3252,7 @@
       [ "stick", 2 ],
       [ "broom", 2 ],
       [ "mop", 2 ],
-      [ "2x4", 1 ],
+      [ "2x4", 2 ],
```

Comment:
That's the ORIGINAL behavior, so you should be asking that question to whoever did that. I changed it from requiring a single 2x4 to needing 2. >_>